src/suricata.c

```diff
@@ -444,6 +448,27 @@ static void GlobalsDestroy(SCInstance *suri)
     suri->pid_filename = NULL;
 }
 
+/**
+ * \brief Used to send OS specific notification of running threads
+ *
+ * \retval TmEcode TM_ECODE_OK on success; TM_ECODE_FAILED on failure.
+ */
+static TmEcode SendOSSpecificNotification(void)
```

Comment:
Suggest naming this function `OnNotifyRunning` or similar. The current name is somewhat generic for a function that performs a very specific task.